R/DESCRIPTION

```diff
@@ -0,0 +1,46 @@
+Package: metaflow
+Type: Package
+Title: R Binding for Metaflow 
+Version: 2.1.0
+Author: Jason Ge [aut], 
+  Savin Goyal [aut, cre]
+Maintainer: Jason Ge <jge@netflix.com>
```

Comment:
Can we set the email to `help@metaflow.org`. That will make is easier to provide visibility to the team about any comms with CRAN maintainers. We can add individual emails in the Author field above.

Reply:
yep will set the email to help@metaflow.org.